src/ImageRunModal.scss

```diff
@@ -27,6 +27,18 @@
         height: calc(var(--pf-global--FontSize--md) * var(--pf-global--LineHeight--md) + 12px);
         align-content: center;
     }
+
+    .pf-c-form__field-group-body {
+        // Stretch content fully
+        --pf-c-form__field-group-body--GridColumn: 1 / -1;
+        // Reduce padding at the top
+        --pf-c-form__field-group-body--PaddingTop: var(--pf-global--spacer--xs);
+    }
+
+    .pf-l-grid {
+        // Rework grid to fix extra spacing between content and removal button
+        grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
+    }
```

Comment:
Not really. This is different. That uses a Bootstrap-like 12 grid placeholder; this doesn't.

As a result of the grid, there's a weird gap between the items and the remove button, which this removes.

Reply:
This is how it looks with fixing of the `pf-m-*-col-on-md`:
![Screenshot from 2022-07-26 08-28-42](https://user-images.githubusercontent.com/12330670/180938851-af774601-6747-4336-beb0-d227d19bcff2.png)

And this is how it looks like in this PR:
![Screenshot from 2022-07-26 08-29-53](https://user-images.githubusercontent.com/12330670/180938915-477ccc95-f39d-41e5-bd08-757a6eca4898.png)

The gap between items and the remove button seems the same in both cases. The difference is that I have changed proportions of some inputs - the most important is the host path which now does not overflow since it has enough space.

I pushed it into https://github.com/cockpit-project/cockpit-podman/pull/1056 where I picked the first rule from this PR.

Reply:
I guess the numbering approach @ https://github.com/cockpit-project/cockpit-podman/pull/1056#issuecomment-1195082163, with a bit of the CSS in this PR, helps. :+1: